.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.1.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
```

Comment:
I went down a little bit of a rabbit hole learning about the `changelog` option. Here's what I found:

The [changesets docs](https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#changelog-false-or-a-path) say:

> As well as the default one, you can use `@changesets/changelog-git`, which adds links to commits into changelogs, or `@changesets/changelog-github`, which requires github authentication, and includes a thankyou message to the person who added the changeset as well as a link to the relevant PR.

At first glance, that makes it sound like `@changesets/cli/changelog` doesn't add links to commits, which I know has been a pain point in some of our older changesets-powered changelogs ([example](https://github.com/getnacelle/compatibility-connector/blob/main/CHANGELOG.md#nacellecompatibility-connector)).

That said, [changes were made to `@changesets/cli` in February](https://github.com/changesets/changesets/pull/666/files) that make `@changesets/changelog-git` the default. Technically, `@changesets/cli/changelog` is now an alias for `@changesets/changelog-git` as of [`@changesets/cli@2.21.0`](https://github.com/changesets/changesets/pull/750/files#diff-6e2e2a1851648938b325ba84de634407a4e69a644ea61102df15ca4a8a7a9758).

So I think the default should be good, as it will provide [links to commits in the changelog](https://github.com/changesets/changesets/blob/main/packages/changelog-git/CHANGELOG.md#changesetschangelog-git). At this point, the only question is whether or not we see value in including a thank you message to contributors in the changelog, [which `@changesets/changelog-github` accomplishes](https://github.com/changesets/changesets/blob/main/packages/changelog-github/CHANGELOG.md#changesetschangelog-github):

<img width="800" alt="changesets changlog github pr thanks to contributors" src="https://user-images.githubusercontent.com/5732000/180514626-c0862c01-9f7e-4632-be43-f8ccb4b766ae.png">